src/handlers/GraphQLHandler.ts

```diff
@@ -150,7 +150,7 @@ Consider naming this operation or using "graphql.operation" request handler to i
     )
   }
 
-  protected override extendInfo(
+  protected extendInfo(
```

Comment:
`override` is good and all but we need to keep in mind it didn't exist prior to `4.5`.